backend/api/src/test/kotlin/software/shonk/adapters/incoming/ShorkInterpreterControllerV1IT.kt

```diff
@@ -57,4 +54,29 @@ class ShorkInterpreterControllerV1IT() : AbstractControllerTest() {
         assertEquals(HttpStatusCode.BadRequest, result.status)
         assert(result.bodyAsText().contains("No player with that name in the lobby"))
     }
+
+    @Test
+    fun `test get lobby status with invalid ID`() = runTest {
+        val result = client.get("/api/v1/lobby/status/1")
+        assertEquals(HttpStatusCode.BadRequest, result.status)
+        assert(result.bodyAsText().contains("No lobby with that id"))
+    }
+
+    @Test
+    fun `test get lobby status with valid (default) ID`() = runTest {
+        val result = client.get("/api/v1/lobby/status/0")
+        val expectedStatus =
+            """
+        {
+            "playerASubmitted": false,
+            "playerBSubmitted": false,
+            "gameState": "NOT_STARTED",
+            "result": {
+                "winner": "UNDECIDED"
+            }
+        }
+        """
+                .trimIndent()
+        assertEquals(expectedStatus, result.bodyAsText())
+    }
```

Comment:
There is no test testing the state of a modified lobby.
There should be one, what if the api always returns the default?

Reply:
I added a new Test for a custom ID using the post endpoint from apv0 (until the new submit code is implemented)